crud.py

```diff
@@ -0,0 +1,49 @@
+from sqlalchemy.orm import Session
+from models import Author, Book
+from schemas import AuthorCreate, BookCreate
+
+
+def get_author_by_name(db: Session, name: str):
+    return (
+        db.query(Author).filter(Author.name == name).first()
+    )
+
```

Comment:
Aren`t we filtering by author id?
![image](https://github.com/mate-academy/py-fastapi-library-management-api/assets/90575903/d52d724f-6654-4c72-b548-617885aae07d)